<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="MapScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>531</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>529</width>
       <height>529</height>
      </rect>
     </property>
     <widget class="QGraphicsView" name="mapView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>530</width>
        <height>530</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TimeBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="travelTimeLcd">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="travelTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Elapsed Time</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ButtonBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>151</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="endTurn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>END TURN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="quitGame">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>QUIT GAME</string>
     </property>
    </widget>
   </widget>
   <widget class="QColumnView" name="statsView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="PortraitView">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNysse">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionGame"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionRules"/>
    <addaction name="actionAbout_Nysse"/>
   </widget>
   <addaction name="menuNysse"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGame">
   <property name="text">
    <string>Scoreboard</string>
   </property>
  </action>
  <action name="actionRules">
   <property name="text">
    <string>Rules</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Nysse">
   <property name="text">
    <string>About Nysse</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
